src/ARCTokenization/AnnotationTable.fs

```diff
@@ -1,4 +1,4 @@
-﻿namespace ArcGraphModel
+﻿namespace ARCTokenization
```

Comment:
if you look at [the file directly](https://github.com/nfdi4plants/ARCTokenization/blob/8006f214444ef203b9f6f870a51a2cb96fd2592a/src/ARCTokenization/AnnotationTable.fs) it is not there. Since the project builds and tests pass, this must be a visual bug on github, since i am absolutely sure this would not compile.